frontend/src/pages/SecondStep/index.tsx

```diff
@@ -36,58 +22,53 @@ type Props = {
 
 const SecondStep: FC<Props> = ({ setSecondStep, secondStep, firstStep }) => {
   const router = useRouter();
-  const classes = useStyles();
 
   const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
     setSecondStep(event.target.value as string);
   };
   return (
     <Wrap>
-      <StyleWrapper margin="3rem 0 1rem 0">
-        <H2>Jaký předmět chcete učit?</H2>
+      <CustomStepper currentStep={2} />
+      <StyleWrapper margin="0 0 2rem 0">
+        <H3>Jaký předmět chcete učit?</H3>
       </StyleWrapper>
-      <PrimaryText>
+      <PrimaryText size="1rem">
         Pro{' '}
         {firstStep.chooseDegreeState.map((button, idx) => {
           if (button.checked) {
             return button.label;
           }
         })}
       </PrimaryText>
-      <StyleWrapper margin="0 0 3rem 0">
-        <LightText>
-          Vyberte si prosím pouze jeden předmět. V případě, že chcete učit i
-          nějaký další předmět, je potřeba vybrat z daných kombinací studijních
-          dvouoborů. Pokud vám ani jedna nebude vyhovovat, můžete se na výuce
-          druhého předmětu domluvit až přímo se svým zaměstnavatelem: ředitelem
-          školy.
-        </LightText>
+      <StyleWrapper margin="0 0 1rem 0">
+        <LightText>Vyberte si prosím pouze jeden předmět.</LightText>
+      </StyleWrapper>
+      <StyleWrapper margin="0 0 1rem 0">
+        <Hint hintText="Chcete učit více předmětů?" />
+        <Hint hintText="Zjistěte, jaké předměty můžete s vaším vzděláním vyučovat" />
       </StyleWrapper>
-      <StyleWrapper margin="2rem 0"></StyleWrapper>
-      <FormControl className={classes.formControl}>
-        {/* <InputLabel id="demo-simple-select-label">Vyberte předmět</InputLabel> */}
-        <Select
-          labelId="demo-simple-select-label"
-          id="demo-simple-select"
-          value={secondStep}
-          onChange={handleChange}
-        >
+      <form>
+        <Select name="subject" id="subject" autoFocus aria-label="Subject">
           {subjects.map((subject) => (
-            <MenuItem value={subject}>{subject}</MenuItem>
+            <option value={subject}>{subject}</option>
           ))}
         </Select>
-      </FormControl>
+      </form>
       <StyleWrapper textAlign="center">
         <Input
           value="Pokračovat   >"
           type="button"
-          padding="1.5rem 2rem"
-          margin="2.5rem 0 0 0"
+          padding="1rem 2rem"
+          margin="2.5rem 0 1rem 0"
+          bgColor={theme.color.primary}
           onClick={() => {
             router.push('/vyber-vzdelani');
           }}
         />
       </StyleWrapper>
+      <StyleWrapper textAlign="center" padding="0 0 2rem 0">
+        <Hint hintText="Nevíte si rady s výběrem předmětu? Napište nám" />
```

Comment:
myslim že tohle jsi nedělala ty. Ale tady by ten text mohl být jako `children` a nebo jenom `<Hint text="...." />`. Neni potřeba aby se to jmenovalo `hintText`, když je to `Hint` komponenta